netkan/netkan/cli.py

```diff
@@ -1,17 +1,26 @@
 import click
-import logging
 import boto3
+import json
+import logging
+import time
 from pathlib import Path
 from .utils import init_repo, init_ssh
 from .github import GitHubPR
 from .indexer import MessageHandler
 from .scheduler import NetkanScheduler
+from .status import ModStatus
+
 
 @click.group()
 def netkan():
     pass
 
+
 @click.command()
+@click.option(
+    '--status-db', envvar='STATUS_DB',
```

Comment:
If I'm understanding `@click.option` correctly, this should be after the `--key` option to match up with the order of the parameters.

Reply:
The order doesn't matter. These are passed through as `kwargs`, so aside from being neat it doesn't matter. There is a lot of duplication going on, but that's a  future thing that can be cleaned up.